index.html

```diff
@@ -1,12 +1,147 @@
+            </ul>
+           </div>
+        </div>
     
+    <div class="mobile-menu" id="mobileMenu">
+        <img id="menu-close" src="images/closeIcon.png" alt="close icon">
+        <ul class="mob-menu">
+          <li class="menu"><a class="moblink-menu" href="./index.html">Home</a></li>
+          <li class="menu"><a class="moblink-menu" href="./about.html">About</a></li>
+          <li class="menu"><a class="moblink-menu" href="#about.html">Program</a></li>
+          <li class="menu"><a class="moblink-menu" href="#contact-me">Join</a></li>
+          <li class="menu"><a class="moblink-menu" href="#about-me">Sponser</a></li>
+          <li class="menu"><a class="moblink-menu" href="#contact-me">Register</a></li>
+        </ul>
+    </div>
+    
+     </div>
+  </nav>
+    <div class="front">
+        <div class="front1">
+        <h2>"hello! developers"</h2>
+        <h1>SOFTWARE DEVELOPERS <br> GLOBAL SUMMIT 2023</h1>
+        </div>
+        <div class="front2">
+        <p class="pbox1">A motivational Conference for those who want to be profecional software developer held by Technology for Africa is taking place in June Ethiopia.</p>
+        </div>
+        <div class="front3">
+        <h3 class="date">2023.10.15(THO) ~ 16(FRI)</h3>
+        <p>@AU Assemble Hall, ADDIS ABABA ETHIOPIA</p>
+        </div>
+    </div>
+</section>
+</header>
+    <section class="mainpage">
+        <h1>Main Page</h1>
+         <hr class="redline">
+        <div class="boxes">
+        <div class="box">
+            <div class="photo"><img src="images/images (4).png" alt=""></div>
+           <h3>Lecture</h3>
+           <p>Listen from the speakers those who came from all over the world and who are profeciaonal in software development </p>
+        </div>
+        <div class="box">
+            <div class="photo"><img src="images/images (1).png" alt=""></div>
+            <h3>CC Exhibition </h3>
+            <p>Appreciate various creations applying software development licens of ariists organized from Art Center Nabi</p>
+        </div>
+        <div class="box">
+            <div class="photo"><img src="images/images (2).png" alt=""></div>
+            <h3>Forum</h3>
+            <p>Have the same time to share to your thoughts, opinions with experts of software developers topic.</p>
+        </div>
+        <div class="box">
+            <div class="photo"><img src="images/images (3).png" alt=""></div>
+            <h3>Workshop</h3> 
+            <p>Try Creating your own apps using open source license rather than just watching about software app.</p>
+        </div>
+        <div class="box">
+            <div class="photo"><img src="images/images.jpg" alt=""></div>
+            <h3>CC Ignite</h3>
+            <p>Get the  chance to network with CC affilliates around the world ,also after the summit of development</p>
+        </div>
+     </div>
+     <div class="redbox"> <h3> <a href="#">View full Program></a> </h3> </div>
+    </section>
```

Comment:
**OPTIONAL** - This section of your design has a few differences from what is in the design template. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

Your project (Please pay attention to the following optional points):
- The heading for this page is **Main Program** not **Main Page**. Please fix this 🙏 
- The color of the header text has color hex code #fff, and the divider (underline) is too faint in your design. Please fix this 👍 👍 
- For semantic reasons, I advise you use the `article` tag for individual program cards. Please fix this 👍 👍 
- The background for this section is different from what is in the design template. Please fix this 😄 
- The bottom text says **See full program** not **view full program** please fix this 🙏 

Your Project:
<img width="948" alt="main program olod" src="https://user-images.githubusercontent.com/73607512/198667314-eeee4f2a-756f-4ce9-a6fd-a734e6219f72.PNG">

Design Template:
<img width="626" alt="main program behance" src="https://user-images.githubusercontent.com/73607512/198667456-29c4586f-a393-4aec-b873-518e9ee5d1fb.PNG">